core/scheduler/src/main/resources/application.conf

```diff
@@ -64,6 +64,11 @@ whisk {
     grpc {
       tls = "false"
     }
+    scheduling {
```

Comment:
since this is already under `scheduler` I think this isn't providing any more descriptiveness. How about calling the config `decision-maker`?